OptionsResolver add normalizer with force prepend error

symfonyaml opened the pull request:
| Q             | A
| ------------- | ---
| Branch?       |  4.4, 5.0
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| License       | MIT

**Issue:**
OptionsResolver::addNormalizer() returns this error when prepend option set.
`Warning: array_unshift() expects parameter 1 to be array, null given`

**Steps to reproduce:**
Execute this form:
```php
use Symfony\Component\Form\AbstractType;
use Symfony\Component\Form\FormBuilderInterface;
use Symfony\Component\OptionsResolver\OptionsResolver;

class TestType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder->add('test');
    }

    public function configureOptions(OptionsResolver $resolver)
    {
        $resolver->setDefault('test', 1);

        $resolver->addNormalizer(
            'test',
            function (OptionsResolver $options, $test) {
                return $test;
            },
            true //  <-------------- $forcePrepend = true
        );
    }
}
```

symfonyaml committed:
OptionsResolver add normalizer with force prepend error

error: Warning: array_unshift() expects parameter 1 to be array, null given

stof reviewed (changes requested):
Please add a test to prevent regressions

symfonyaml committed:
Tests OptionsResolver add normalizer prepend

symfonyaml committed:
Apply fabbot.io standards suggestions

Apply https://fabbot.io/patch/symfony/symfony/36339/7b67dd508bbfa7953b55b81e61b8c3a522a63944/exception_messages.diff

Commit:
Revert "Apply fabbot.io standards suggestions"

This reverts commit 467018d8be5c44e5c5a42ad60e6e1eda1906e9cd.

fabpot commented:
already fixed by 6eb836b6f9b98c3c1b3078af6afd14c8ab3296e4
